src/1.21.8/main/resources/assets/skyblock-pv/shaders/text/gradient.fsh

```diff
@@ -47,7 +42,18 @@ void main() {
     if (color.a < 0.1) {
         discard;
     }
-    vec2 coords = gl_FragCoord.xy;
+    vec4 finalColor = vertexColor;
+
+    if (finalColor.r != 0.0 || finalColor.g != 0.0 || finalColor.b != 0.0) {
```

Comment:
Isn't this equal to `len(finalColor.rgb) != 0` (idk if its len but smth like that)